packages/nextjs/components/CheckedInChart.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import { CategoryScale, Chart as ChartJS, LineElement, LinearScale, PointElement, TimeScale, Tooltip } from "chart.js";
+import { Line } from "react-chartjs-2";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+export const CheckedInChart = () => {
+  ChartJS.register(CategoryScale, LinearScale, TimeScale, PointElement, LineElement, Tooltip);
+
+  const { data: checkedInEvents, isLoading } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 118484927n,
+    watch: true,
```

Comment:
I think we can remove this. It's going to cause a lot of RPC calls... and we don't need to be that dynamic in this case :D

Reply:
Also I think is causing to be re-rendering every 30s (because of the RPC polling interval)